CONTRIBUTING.md

```diff
@@ -3,3 +3,21 @@
 Our contributor guidelines are available on the public wiki here:
 https://wiki.crdb.io/wiki/spaces/CRDB/pages/73204033/Contributing+to+CockroachDB
 
+At this location, we share our team guidelines and knowledge
+base regarding:
+
+- repository layout
+- how to organize your code change
+- commenting guidelines
+- commit message guidelines
+- how to write and run tests
+- how to write release notes
+- how to submit a change for review
+- how to use continuous integration (CI)
+- how to troubleshoot certain issues
+
+as well as many other practical topics.
+
+If you have any question, hop into our [CockroachDB Community
+Slack](https://cockroa.ch/slack) where you can ask questions to the
+CockroachDB team directly.
```

Comment:
should we call out the #contributors channel?

Reply:
sure done